packages/connection/src/Client.php

```diff
@@ -45,7 +45,8 @@ public static function remote_request( $args, $body = null ) {
 			$args['auth_location'] = 'query_string';
 		}
 
-		$token = \Jetpack_Data::get_access_token( $args['user_id'] );
+		$connection = new Manager();
```

Comment:
not from this PR, but I am not a fan of the "Manager" name 😞

Reply:
Would `Jetpack::connection()` be less evil here?

Reply:
> Would Jetpack::connection() be less evil here?

yes +1

Reply:
Yeah, me too. But that's the name of the class, and we're in the same namespace, so 🤷‍♂

Reply:
We're trying to decouple away from anything that's outside of the package. What's the problem with instantiating this, we're not doing anything on `__construct()`?

Reply:
> What's the problem with instantiating this
I wouldn't say that there's a problem, I'm just trying to figure out the best way to get the instance here. I think we agreed that it's fine to instantiate, because most of the state is stored in the options anyway. If you feel like this would be a better way to go, let's keep this as is for now.

Reply:
Nah, I don't insist on instantiating - just if we use `Jetpack::connection()` here, we'll lose the purpose of this PR, which is decoupling. Happy to go the way you folks suggest.

Reply:
Let me know if you prefer closing the PR instead.

Reply:
No, it's fine, let's keep this way here, you make a good point about decoupling - we shouldn't be going back to the Jetpack class for this object.

Reply:
Thanks for bringing it up anyway, it was a worthy discussion!